hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsOutputStream.java

```diff
@@ -287,13 +297,13 @@ private synchronized void writeCurrentBufferToService() throws IOException {
       @Override
       public Void call() throws Exception {
         client.append(path, offset, bytes, 0,
```

Comment:
In the trunk, client calls have AbfsPerfTracker tracking all the calls. Not sure why the PR view is failing to show it as a delta from trunk. But looks like there is some issue here with merge from trunk. Please do a file to file compare from PR branch to trunk.
example: from trunk:
`      public Void call() throws Exception {
        AbfsPerfTracker tracker = client.getAbfsPerfTracker();
        try (AbfsPerfInfo perfInfo = new AbfsPerfInfo(tracker,
                "writeCurrentBufferToService", "append")) {
          AbfsRestOperation op = client.append(path, offset, bytes, 0,
                  bytesLength);
          perfInfo.registerResult(op.getResult());
          byteBufferPool.putBuffer(ByteBuffer.wrap(bytes));
          perfInfo.registerSuccess(true);
          return null;
        }`
(this is not the only delta)